src/main/java/teammates/ui/pagedata/InstructorHomeCourseAjaxPageData.java

```diff
@@ -128,8 +128,10 @@ private void addAttributeIf(boolean shouldAdd, ElementTag button, String key, St
                     getInstructorSubmissionStatusForFeedbackSession(session),
                     getInstructorPublishedStatusForFeedbackSession(session),
                     TimeHelper.formatDateTimeForInstructorHomePage(session.getStartTimeLocal()),
+                    session.getStartTimeInIso8601UtcFormat(),
                     TimeHelper.formatDateTimeForSessionsFixedOffset(session.getStartTime(), session.getTimeZone()),
```

Comment:
Oh yes, can change this back to `formatDateTimeForSessions` and delete this method from `TimeHelper`? It's a workaround for this issue.